docs/develop/extended-overrides.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Work with extended overrides of the manifest
+description: 'Learn how to configure extensibility features with extended overrides of the manifest.'
+ms.date: 01/30/2021
+localization_priority: Normal
+---
+
+# Work with Extended Overrides of the manifest
+
+Some extensibility features of Office Add-ins are configured with JSON files that are hosted on your server, instead of with the add-in's XML manifest.
+
+> [!NOTE]
+> This article assumes that you are familiar with Office add-in manifests and their role in add-ins. Please read [Office Add-ins XML manifest](add-in-manifests.md), if you have not done so recently.
+
+The following table specifies the extensibility features that require an extended override along with links to documentation of the feature.
+
+| Feature | Development Instructions |
+| :----- | :----- |
+| Keyboard shortcuts | [Add Custom keyboard shortcuts to your Office Add-ins](../design/keyboard-shortcuts.md) |
+
+The schema that defines the JSON format is [extended-manifest schema](https://developer.microsoft.com/json-schemas/office-js/extended-manifest.schema.json).
+
+> [!TIP]
+> This article is somewhat abstract. Consider reading one of the articles in the table to give concreteness to the concepts.
+
+## Tell Office where to find the JSON file
+
+Use the manifest to tell Office where to find the JSON file. Immediately *below* (not inside) the `<VersionOverrides>` element in the manifest, add an [ExtendedOverrides](../reference/manifest/extendedoverrides.md) element. Set the `Url` attribute to the full URL of a JSON file. The following is an example of the simplest possible `<ExtendedOverrides>` element.
+
+```xml
+    ...
+    </VersionOverrides>  
+    <ExtendedOverrides Url="https://contoso.com/addin/extended-overrides.json"></ExtendedOverrides>
+</OfficeApp>
+```
+
+The following is an example of a very simple extended overrides JSON file. It assigns keyboard shortcut CTRL+SHIFT+A to a function (defined elsewhere) that opens the add-in's task pane.
+
+```json
+{
+    "actions": [
+        {
+            "id": "SHOWTASKPANE",
+            "type": "ExecuteFunction",
+            "name": "Show task pane for add-in"
+        }
+    ],
+    "shortcuts": [
+        {
+            "action": "SHOWTASKPANE",
+            "key": {
+                "default": "CTRL+SHIFT+A"
+            }
+        }
+    ]
+}
+```
+
+## Localizing the extended overrides file
+
+If your add-in supports multiple locales, you can use the `ResourceUrl` attribute of the `<ExtendedOverrides>` element to point Office to a file of localized resources. The following is an example.
```

Comment:
```suggestion
If your add-in supports multiple locales, use the `ResourceUrl` attribute of the `<ExtendedOverrides>` element to point Office to a file of localized resources. The following is an example.
```
Microsoft Voice & Style guidelines: "Kill off weak writing. Most of the time, start each sentence with a verb. Cut "you can" and "there is", “there are”, “there were.” Consider reading through these topics again and remove them where _you can_. [Microsoft Voice guidelines](https://microsoft.sharepoint.com/teams/BrandCentral/Pages/The-Microsoft-brand-Identity-Voice.aspx)

Reply:
The problem with replacing "You can do x" with "Do x" is that the latter sounds like a requirement, whereas the "You can" conveys that it is an option. This sentence is a case where it is an option. (There's another way to localize with multiple locales described later in the article.)

Reply:
Clearly pointed out in our guidance... we are not to use "you can" if possible.  The recommendation is to use our voice to provide strong guidance. The "you can" is implied. Please remove them all to conform to the Microsoft Voice and Style.